spec/changelog.yml

```diff
@@ -28,11 +28,9 @@
   - description: Make exception for legacy input controls
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/619
-- version: 2.13.1-next
-  changes:
-  - description: Prepare for next version
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/623
+  - description: Update import paths for Go code
+    type: breaking-change
+    link: https://github.com/elastic/package-spec/issues/626
```

Comment:
This changelog is still in `3.0.0-next`. Another PR would be needed to change it to `3.0.0-rc1`

Reply:
Yep, we can do the usual "Prepare version..." PR after merging this one.